projects/laji/src/app/+project-form/project-form.component.ts

```diff
@@ -245,4 +255,9 @@ export class ProjectFormComponent implements OnInit, OnDestroy {
   trackByLabel(index, link) {
     return link.label;
   }
+
+  checkToggledSidebar(event) {
+    this.openSidebar = event;
+    this.showNav$ = of(event);
+  }
```

Comment:
Is it necessary to know whether the sidebar has been toggled? There could be some reason I'm not seeing. I'm pretty sure this is the reason for the following bug:

1. Go to named places view in desktop mode
2. Close sidebar
3. Click some place
4. Sidebar opens

Reply:
That idea to check if sidebar is open/closed was a little confused. But the problem is in sidebar component when it's setting "open" value. As in mobile ShowNav is always false and it never changes, the "toggled" emitter doesn't emit